warhammer-core/WarhammerCore.WebApi/Controllers/UserController.cs

```diff
@@ -0,0 +1,54 @@
+﻿using Microsoft.AspNetCore.Authorization;
+using Microsoft.AspNetCore.Mvc;
+using System;
+using System.Collections.Generic;
+using System.Threading.Tasks;
+using WarhammerCore.Abstract.Exceptions;
+using WarhammerCore.Abstract.Interfaces;
+using WarhammerCore.Abstract.Models;
+using WarhammerCore.WebApi.Models.Enums;
+using WarhammerCore.WebApi.Models.Request;
+using WarhammerCore.WebApi.Models.Response;
+
+namespace WarhammerCore.WebApi.Controllers
+{
+    public class UserController : ApiControllerBase
+    {
+        private readonly IUserService _userService;
+
+        public UserController(IUserService userService)
+        {
+            _userService = userService ?? throw new ArgumentNullException(nameof(userService));
+        }
+
+        [AllowAnonymous]
+        [HttpPost]
+        public async Task<ActionResult<SigninResponse>> Signin(SigninRequest request)
+        {
+            var token = await _userService.SignInAsync(request.Email, request.Password);
+
+            if(token == null)
+            {
+                return NotFound(new ErrorResponse(ErrorCode.UserNotFound));
+            }
+            return new SigninResponse { Email = request.Email, Token = token.Token };
+        }
+        [Authorize(AuthenticationSchemes = JwtBearerDefaults.AuthenticationScheme)]
```

Comment:
This method needs `[AllowAnonymous]` attribute, otherwise we get an 401.

![image](https://user-images.githubusercontent.com/43864376/104847727-2ade5880-58e2-11eb-9288-64105125f37b.png)

```suggestion
        [AllowAnonymous]
```